jelares/portfolio/target/portfolio-1/javascript/script.js

```diff
@@ -26,4 +26,11 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
+} 
+
+// window.addEventListener('scroll', function(e) {
+
+//     const target = document.querySelectorAll('.parallax');
+//     console.log(target.style);
+
+// });
```

Comment:
Can this file be deleted now? You'll still be able to retrieve it from source control if needed in the future.

Reply:
Updated to ignore target